tests/ignite/metrics/test_roc_curve.py

```diff
@@ -49,9 +49,11 @@ def test_roc_curve(available_device):
     sk_fpr, sk_tpr, sk_thresholds = roc_curve(
         to_float32_if_mps(np_y, available_device), to_float32_if_mps(np_y_pred, available_device)
     )
-    sk_fpr = to_float32_if_mps(sk_fpr, available_device)
-    sk_tpr = to_float32_if_mps(sk_tpr, available_device)
-    sk_thresholds = to_float32_if_mps(sk_thresholds, available_device)
+
```

Comment:
Let's create np_y_pred and np_y with torch.rand and torch.zeros in this test as well

Reply:
@BanzaiTokyo check this comment ^^^